.github/workflows/pr-label-milestone-check.yml

```
@@ -9,6 +9,11 @@ jobs:
     runs-on: ubuntu-latest
```

Need to update this runner to be the self-hosted runner.

```suggestion
    runs-on: smart-contracts-linux-medium
```